diff -Naur broadcom-sta-6.30.223.248.orig/amd64/src/wl/sys/wl_cfg80211_hybrid.c broadcom-sta-6.30.223.248/amd64/src/wl/sys/wl_cfg80211_hybrid.c
--- broadcom-sta-6.30.223.248.orig/amd64/src/wl/sys/wl_cfg80211_hybrid.c	2015-03-21 19:54:54.400827535 +0700
+++ broadcom-sta-6.30.223.248/amd64/src/wl/sys/wl_cfg80211_hybrid.c	2015-03-21 20:28:54.849230399 +0700
@@ -2010,7 +2010,11 @@
 
 	notify_ie = (u8 *)bi + le16_to_cpu(bi->ie_offset);
 	notify_ielen = le32_to_cpu(bi->ie_length);
-	cbss = cfg80211_inform_bss(wiphy, channel, (const u8 *)(bi->BSSID.octet),
+	cbss = cfg80211_inform_bss(wiphy, channel, 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 18, 0)
+  CFG80211_BSS_FTYPE_UNKNOWN,
+#endif
+    (const u8 *)(bi->BSSID.octet),
 		0, beacon_proberesp->capab_info, beacon_proberesp->beacon_int,
 		(const u8 *)notify_ie, notify_ielen, signal, GFP_KERNEL);
