diff -Naur broadcom-sta-6.30.223.248.orig/amd64/src/wl/sys/wl_linux.c broadcom-sta-6.30.223.248/amd64/src/wl/sys/wl_linux.c
--- broadcom-sta-6.30.223.248.orig/amd64/src/wl/sys/wl_linux.c	2015-03-21 19:54:54.399827540 +0700
+++ broadcom-sta-6.30.223.248/amd64/src/wl/sys/wl_linux.c	2015-03-21 20:19:18.415021682 +0700
@@ -2156,9 +2156,9 @@
 
 	wlif = WL_DEV_IF(dev);
 	wl = WL_INFO(dev);
-
+  
+  skb->prev = NULL;
 	if (WL_ALL_PASSIVE_ENAB(wl) || (WL_RTR() && WL_CONFIG_SMP())) {
-		skb->prev = NULL;
 
 		TXQ_LOCK(wl);
 
